x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -21,27 +21,15 @@ import { TelemetryDiagTask } from './diagnostic_task';
 import { TelemetryEndpointTask } from './endpoint_task';
 import { EndpointAppContextService } from '../../endpoint/endpoint_app_context_services';
 import { AgentService, AgentPolicyServiceInterface } from '../../../../fleet/server';
+import {
```

Comment:
```sh
node scripts/precommit_hook --fix
```

to fix your code styling

Reply:
Hmm no output or files modified when run?

Reply:
You need to run `git add && git commit -m ""` first